lib/identity_config.rb

```diff
@@ -371,6 +371,10 @@ def self.store
     config.add(:show_user_attribute_deprecation_warnings, type: :boolean)
     config.add(:short_term_phone_otp_max_attempts, type: :integer)
     config.add(:short_term_phone_otp_max_attempt_window_in_seconds, type: :integer)
+    config.add(:sign_in_user_id_per_ip_attempt_window_exponential_factor, type: :float)
+    config.add(:sign_in_user_id_per_ip_attempt_window_in_minutes, type: :integer)
+    config.add(:sign_in_user_id_per_ip_attempt_window_max_days, type: :integer)
```

Comment:
I think using hours or even minutes for finer-grained control would be preferable if in the future we need to do part of a day for some reason.